validator/testcases/installrdf.py

```diff
@@ -93,6 +94,21 @@ def _test_rdf(err, install):
         elif predicate == 'multiprocessCompatible':
             err.save_resource('is_multiprocess_compatible', value == 'true')
 
+        elif predicate == 'hasEmbeddedWebExtension':
+            err.notice(('testcases_installrdf',
+                        '_test_rdf',
+                        'hasEmbeddedWebExtension'),
+                       'This add-on is an hybrid add-on',
```

Comment:
@wagnerand yeah, it is definitely better to keep it consistent to the terminology used on MDN.

updated patch is coming